discovery.md

```diff
@@ -432,78 +367,105 @@ requirements for the query API.
     the `datacontenttype`.
   - If `dataschama` is present, this field MUST NOT be present.
 
-###### protocols
-
-- Type: `List` of `String`
-- Description: This field describes the different values that might be passed
-  in the `protocol` field of the CloudSubscriptions API. The protocols with
-  existing CloudEvents bindings are identified as AMQP, MQTT3, MQTT5, HTTP,
-  KAFKA, and NATS. An implementation MAY add support for further
-  protocols. All producers MUST support at least one delivery protocol, and MAY
-  support additional protocols.
-- Constraints:
-  - REQUIRED
-  - MUST be non-empty.
-- Examples:
-  - ["HTTP"]
-  - ["HTTP", "AMQP", "KAFKA"]
-
 ###### extensions
 
 - Type: `Map` of `String` to `String`
-- Description: Associative map of CloudEvents [Extension Context Attributes](https://github.com/cloudevents/spec/blob/master/spec.md#extension-context-attributes)
+- Description: Associative map of CloudEvents
+  [Extension Context Attributes](https://github.com/cloudevents/spec/blob/master/spec.md#extension-context-attributes)
   that are used for this event `type`. Keys MUST be confirm to the extension
   context attributes naming rules and value are the type of the extension
   attribute, conforming to the CloudEvents [type system](./spec.md#type-system).
 - Constraints:
   - OPTIONAL
```

Comment:
Please add examples here as well. 
Did you imagine something like
- "dataref" (from [Dataref extension](https://github.com/cloudevents/spec/blob/master/extensions/dataref.md))
- "traceparent" (from [Distributed Tracing extension](https://github.com/cloudevents/spec/blob/master/extensions/distributed-tracing.md))
?

Reply:
Done - had to convert it from a map to an array.